common/beerocks/hostapd/CMakeLists.txt

```diff
@@ -0,0 +1,135 @@
+project(prplmseh_hostapd VERSION ${prplmesh_VERSION})
+
+# Set the base path for the current module
+set(MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
+
+# Support more permissive c++ language
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
+
+# Default defaults
+set(PRPLMESH_HOSTAPD_TYPE "DUMMY" CACHE STRING "PRPLMESH_HOSTAPD type")
+set_property(CACHE PRPLMESH_HOSTAPD_TYPE PROPERTY STRINGS "DUMMY" "NL80211" "DWPAL")
+add_definitions(-DBEEROCKS_TMP_PATH="${TMP_PATH}")
+
+if (TARGET_PLATFORM STREQUAL "rdkb")
+
+    set(PRPLMESH_HOSTAPD_TYPE "DWPAL")
+    # Extra libraries
+    list(APPEND PRPLMESH_HOSTAPD_LIBS rt dl)
+
+elseif(TARGET_PLATFORM STREQUAL "openwrt")
+
+    if (TARGET_PLATFORM_TYPE STREQUAL "ugw")
+        set(PRPLMESH_HOSTAPD_TYPE "DWPAL")
+    else()
+        set(PRPLMESH_HOSTAPD_TYPE "NL80211")
+    endif()
+
+    # hostapd directory
+    file(GLOB PRPLMESH_HOSTAPD_SEARCH_PATHS "${PLATFORM_BUILD_DIR}/hostapd*/hostapd-*")
+    find_path(PRPLMESH_HOSTAPD_INCLUDE_DIR NAMES "src/common/wpa_ctrl.h" PATHS ${PRPLMESH_HOSTAPD_SEARCH_PATHS} NO_CMAKE_FIND_ROOT_PATH)
+    set(PRPLMESH_HOSTAPD_DIR "${PRPLMESH_HOSTAPD_INCLUDE_DIR}")
+
+endif()
```

Comment:
None of this is needed. If it turns out to be needed, introduce it when you need it.

Reply:
Removed
Will be introduced when needed.